### DISCUSSION TITLE
Will HEVC be supported with SRT?

### DISCUSSION THREAD

**Discussion by jhays** [HaishinKit/HaishinKit.swift#1424] [Q&A] [answered]
Hello, I wanted to reach out to ask whether there are any plans to support HEVC with SRT.

I'm aware of this discussion:
https://github.com/shogo4405/HaishinKit.swift/issues/1292

But given that this is from last year and several HaishinKit versions back, I wanted to check again to understand whether there are any plans to support HEVC via SRT at some point. 

Thanks

**Comment by shogo4405** [comment, marked as answer]
Unfortunately, I have no plan. I don't think it's difficult, so I hope someone who needs it will contribute.

**Comment by shogo4405** [comment]
I implemented HEVC for SRT in this pull request. It will be released as version 1.8.2. https://github.com/shogo4405/HaishinKit.swift/pull/1464

**Reply by jhays** [reply]
Thank you for the update! I will hopefully test it out soon.
